CHANGELOG.md

```diff
@@ -1,170 +1,17 @@
-- Esri Community - 1056484: Fixed an issue where ImageryLayer was throwing a warning when UniqueValueRenderer and renderingRule are both set on Imagerylayer.
-- Esri Community - 1058789: Fixed an issue where the OGCFeatureLayer was requesting the wrong CRS for some services, which could cause a slight offset.
-- Esri Community - 1058935: Fixed an issue where updates made to a map's FeatureLayer did not reflect within the corresponding FeatureTable widget.
-- Esri Community - 1059410: Fixed an issue where ImageryLayer was throwing an error when lerc format was requested.
-- GitHub - 109 - Fixed an issue with esri-loader throwing unhandled promise exceptions when used with Angular 12 and zone.js.
-- GitHub - 296 - Fixed an issue where sometimes the map would not display when adding a ScaleBar widget to a MapImageLayer without a basemap.
-- Fixed an issue in 2D MapViews where labels would not display if the where clause used a field that wasn't in the labelExpressionInfo.expression.
-- Fixed an issue where a CIMSymbol with CIMPictureMarker symbol layer would not render when the image size was too large.
-- Fixed an issue where a multi-level nested CIMSymbol would not render.
-- Fixed an issue where CIMSymbol's primitiveOverride was not working with the color property of a CIMSolidFill symbol layer.
-- Fixed an issue where features with null geometries could display off the map.
-- Fixed an issue where hitTest would fail on a line CIMSymbol with a VectorMarker symbol layer.
-- Fixed an issue where printing clustered features with different browser window sizes caused inconsistent print results.
-- Fixed an issue where the font color of links in the FeatureTable in dark theme did not have enough contrast with the table's background color.
-- Fixed an issue where the layer fails to draw features when some Arcade geometry functions are used in the renderer.
-- Fixed an issue where the Search widget suggestions dropdown could remain open when a user clicks on another HTML element on the page.
-- Fixed an issue where time enabled layers were not printed properly with the Print widget and PrintTask.
-- Fixed an issue where visual variables were not being honored when set on a DictionaryRenderer.
-- Fixed an issue with the geometryEngine.difference() method failing with some line geometries.
-- Fixed an issue with the LayerList widget behavior when working with non-dynamic MapImageLayer sublayers.
-- Fixed an issue with the z-order of symbol layers in a CIMSymbol.
-- Esri Community - 1009348: Enhanced the CoordinateConversion widget to use sessionStorage with a new storageEnabled property.
-- Enhanced summaryStatistics by adding support for string fields.
-- Enhanced summaryStatistics by adding a nullcount property in SummaryStatisticsResult to return the number of null values stored in a given field.
-- Enhanced SceneLayer by supporting Basis Texture and 3D extent provided by I3S version 1.8.
+TBD
```

Comment:
same thing here - there's already a bug fix
> BUG-000141284: Fixed an issue with WMSLayer where custom parameters would be added to the GetLegendGraphic request with a `?` instead of an `&`, causing the request to fail.